### PR TITLE
add mongo authentication documentation

### PR DESCRIPTION
**Pull request by vincentnegre**
add documentation to activate mongodb authentication

### CODE REVIEW

**Comment by vincentnegre** [comment]
feature needed for remote installation of mongo database

File: docs/MongoDBInstallv4.md [hunk 1]
```diff
@@ -125,6 +125,68 @@ You can check the status of the replica set with the following command:
...
+
+3. Optional - you can test authenfication:
+```javascript
+use admin
+db.auth("superadmin", "mydatabase")
+```
+
+4. Create an admin account for the opensilex database:
+
+The role readWrite allow to :
```
Comment (pierreetiennealary) [inline docs/MongoDBInstallv4.md]:
```suggestion
The role readWrite allow to :

```

File: docs/MongoDBInstallv4.md [hunk 2]
```diff
@@ -125,6 +125,68 @@ You can check the status of the replica set with the following command:
...
+- create index
+
+If the mongo server hosts several opensilex instances you can create one user per database.
+
+```javascript
+db.createUser(
+ {
+   user: "opensilex",
+   pwd: "azerty",
+   roles: [ { role: "readWrite", db: "mydatabase" } ]
```
Comment (pierreetiennealary) [inline docs/MongoDBInstallv4.md]:
```suggestion
   roles: [ { role: "readWrite", db: "opensilex" } ]
```

File: docs/MongoDBInstallv4.md [hunk 3]
```diff
@@ -125,6 +125,68 @@ You can check the status of the replica set with the following command:
...
+ }
+)
+
+```
+Check access :
+
+`mongo -u "opensilex" -p "azerty" --authenticationDatabase "mydatabase" "mydatabase"`
```
Comment (pierreetiennealary) [inline docs/MongoDBInstallv4.md]: Please change the ` to ```

File: docs/MongoDBInstallv4.md [hunk 4]
```diff
@@ -125,6 +125,68 @@ You can check the status of the replica set with the following command:
...
+
 ***
 
 Go back to the parent page, the [OpenSILEX local installation documentation](./MongoDBInstallv4.md#mongodb-and-robo-3t).
```
Comment (pierreetiennealary) [inline docs/MongoDBInstallv4.md]:
```suggestion
Go back to the parent page, the [OpenSILEX local installation documentation](./localInstallation.md#mongodb-and-robo-3t).
```

File: docs/MongoDBInstallv4.md (hunk 3 above)
Comment (vincentnegre) [reply docs/MongoDBInstallv4.md]: done

**Pull request merged**